gcovr/gcov_parser.py

```diff
@@ -1024,16 +1051,67 @@ def _find_excluded_ranges(
                 else:  # pragma: no cover
                     pass
 
+            # branche exclusion
+
+            excl_branch_pattern = re.compile(
+                "(" + exclude_pattern_prefix + ")" + _EXCLUDE_BRANCH_PATTERN_POSTFIX
+            )
+            for header, flag in excl_branch_pattern.findall(code):
+
+                if flag == "LINE":
+                    if exclusion_stack_branch:
+                        warnings.branch_after_start(
```

Comment:
The warning is the same. Replaced by accident?
```suggestion
                        warnings.line_after_start(
```

Reply:
Fixed